src/status_im/subs/wallet/wallet_connect.cljs

```diff
@@ -56,6 +57,22 @@
       (some #(string/includes? % address) accounts))
     sessions)))
 
+(rf/reg-sub
+ :wallet-connect/network-filtered-sessions
+ :<- [:wallet-connect/sessions-for-current-account]
+ :<- [:profile/test-networks-enabled?]
+ (fn [[sessions testnet-mode?]]
+   (filter
+    (fn [session]
+      (let [chain-ids (map (fn [chain]
+                             (-> chain
+                                 (string/split #":")
+                                 second
+                                 js/parseInt))
+                           (:chains session))]
```

Comment:
Is it always consistent? Do chains always follow the same format?

Reply:
Can you elaborate what you mean by this ?